### PR TITLE
register check-markdown command to composer script

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
I've added register check-markdown command to composer script and  removed .github/workflows/check_markdown.yml

File: composer.json [hunk 1]
```diff
@@ -277,7 +277,10 @@
             "phpunit"
         ],
         "check-cs": "vendor/bin/ecs check --ansi",
-        "fix-cs": "vendor/bin/ecs check --fix --ansi",
+        "fix-cs": [
+            "vendor/bin/ecs check --fix --ansi",
+            "vendor/bin/ecs check-markdown README.md"
```
Comment (TomasVotruba) [inline composer.json:282]: I've just noticed.
Does this fix or check? The `--fix` behaviour should be the same as in `check` command

Comment (samsonasik) [reply composer.json:282]: It is for fix. Ok, I will try add `--fix` as well for `check-markdown` and `check-heredoc-nowdoc`

Comment (samsonasik) [reply composer.json]: added `--fix` option, need new tag of `easy-coding-standard` first

Comment (TomasVotruba) [reply composer.json]: Tagged :+1:

**Comment by samsonasik** [comment]
I tried update to latest tagged version and got the following error:

```bash
composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for symplify/easy-coding-standard ^8.2.27 -> satisfiable by symplify/easy-coding-standard[8.2.27].
    - symplify/easy-coding-standard 8.2.27 requires symplify/autowire-array-parameter ^8.2.27 -> satisfiable by symplify/autowire-array-parameter[8.3.x-dev] but these conflict with your requirements or minimum-stability.
```

**Comment by samsonasik** [comment]
I've  update symplify/autowire-array-parameter to ^8.2.27 and symplify/easy-coding-standard to ^8.2.27, It is working ok now 👍 

Screen Shot 2020-09-09 at 10 31 17 PM

Screen Shot 2020-09-09 at 10 30 52 PM

**Comment by TomasVotruba** [comment]
Thank you for both updates :+1:

**Pull request merged**